iOS/DuckDuckGo/TabViewController.swift

```diff
@@ -2859,6 +2860,29 @@ extension TabViewController: SecureVaultManagerDelegate {
         }
     }
     
+    private func presentSaveCreditCardModal(with vault: SecureVaultManager, creditCard: SecureVaultModels.CreditCard) {
```

Comment:
Don't really want to block this PR as it's an architectural issue, but I think we need to address the size and responsibilities of the `TabViewController`. Probably needs some sort of routing component for things like this. But it's a bigger problem than just this PR really.